site/content/docs/5.1/examples/list-groups/list-groups.css

```diff
@@ -34,3 +34,27 @@
   filter: none;
   opacity: .5;
 }
+
+.list-group-radio .list-group-item {
+  cursor: pointer;
+  border-radius: .5rem;
+}
+.list-group-radio .form-check-input {
+  margin-top: -.5em;
+  z-index: 2;
+}
+.list-group-radio .list-group-item:hover {
+  background-color: var(--bs-light);
+}
+
+.list-group-radio .form-check-input:checked + .list-group-item {
+  background-color: var(--bs-body);
+  border-color: var(--bs-blue);
+  box-shadow: 0 0 0 2px var(--bs-blue);
+}
+.list-group-radio .form-check-input[disabled] + .list-group-item,
+.list-group-radio .form-check-input:disabled + .list-group-item {
```

Comment:
Makes me wonder if we really need both selectors, as of now. The related issues (#28247) was about Edge 42.

Reply:
it duplicate selector, but `.disable` is missed:
```scss
.list-group-radio .form-check-input.disabled + .list-group-item
.list-group-radio .form-check-input:disabled + .list-group-item
```

Reply:
Let's leave it for now since we use both elsewhere. We can revisit though in another PR to remove that throughout the codebase perhaps. Or hold until v6 if needed.